guides/dimensiondata.md

```diff
@@ -0,0 +1,721 @@
+
+```java
+    javac -version
+```
+
+* Ensure you are using [Maven version 3](http://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) or later. You can check this by running:
+
+```java
+    mvn -version
+```
+
+Now that you have validated the pre-requisities, you will want to do the following: 
+
+* Create a directory to try out jclouds. This can be done by doing: 
+
+```
+    mkdir jclouds
+
+    cd jclouds
+```
+
+* Make a local copy of the pom.xml file below in the jclouds directory.
+
+```
+    mvn dependency:copy-dependencies "-DoutputDirectory=./lib"
+```
+
+You should now have a directory with the following structure:
+
+```
+    jclouds/
+        pom.xml
+        lib/		
+            *.jar
+```
+
+The Dimension Data CloudControl provider is currently available as part of the jclouds labs project [here](https://github.com/jclouds/jclouds-labs).
+The Dimension Data CloudControl provider examples project is currently available as part of jclouds examples [here](https://github.com/jclouds/jclouds-examples).
+
+### Authentication
+
+Connecting to Dimension Data CloudControl can be done by creating a compute connection with the *dimensiondata-cloudcontrol* provider.
+
+To build an unwrapped version of the API do the following.
+{% highlight java %}
+        ContextBuilder contextBuilder = ContextBuilder.newBuilder("dimensiondata-cloudcontrol");
+        DimensionDataCloudControlApi api = contextBuilder
+                .endpoint(endpoint)
+                .credentials(username, password)
+                .buildApi(DimensionDataCloudControlApi.class);
+{% endhighlight %}
+
+
+**Caution:** You will want to ensure you follow security best practices when using credentials within your code or stored in a file.
+
+## <a id="portable-abstraction-usage"></a>Portable Abstraction Usage
+
+### Terms
+Like any cloud provider, Dimension Data CloudControl has its own set of terms in cloud computing. To abstract this into jclouds' Compute interface, these terms were associated:
+
+- Node - a `Server`
+- Image - both *user-uploaded* and *provided* `CustomerImage` and `OSImage`
+- Location - `Geographic Region` equates to a jclouds Region (AU, NA, EU etc) and `Datacenter` equates to a jclouds Zone (NA9, NA12 etc). There is further information on the Geographic Regions [here](https://docs.mcp-services.net/display/CCD/Introduction+to+Geographic+Regions).
+- Hardware - number of cores, RAM size and storage size for a given Image
+
+### Getting Started
+
+{% highlight java %}
+    ComputeService compute = ContextBuilder.newBuilder( "dimensiondata-cloudcontrol" )
+                             .credentials( "username", "password" )
+                             .buildView( ComputeServiceContext.class )
+                             .getComputeService();
+{% endhighlight %}                             
```

Comment:
AFAIK there is no `ComputeSerrvice` implementation yet. Does this work?

Reply:
This is not yet implemented. I am going to leave a note instead to use the unrwapped version of the API for now.